config.md

```diff
@@ -394,6 +403,16 @@ The default is `FullPlaceholders`.
 This option governs the completion of template names as well, where
 the delimiters are `<>`.
 
+### HeaderInsertion
+{:.v20}
```

Comment:
This config option will actually first appear in clangd 21. (llvm 20 has branched in January, so any commits more recent than that target llvm 21 unless they are explicitly backported to the llvm 20 branch.)

Reply:
Ah I didn't realize, 20 had already released, thanks! Should be fixed now.